package.json

```diff
@@ -19,10 +20,12 @@
   "license": "ISC",
   "devDependencies": {
     "husky": "^9.1.7",
+    "nodemon": "^3.1.10",
     "sass": "^1.60.0"
   },
   "dependencies": {
     "govuk-frontend": "^5.9.0",
+    "http-server": "^14.1.1",
```

Comment:
Would it be better to just have this as devDependency as its only used when doing `npm run dev`?

Reply:
Yes! Good spot, thank you. I have fixed it.